src/excalidraw-app/data/firebase.ts

```diff
@@ -106,7 +110,14 @@ const encryptElements = async (
   key: string,
   elements: readonly ExcalidrawElement[],
 ): Promise<{ ciphertext: ArrayBuffer; iv: Uint8Array }> => {
-  const json = JSON.stringify(elements);
+  const json = JSON.stringify(
+    elements.filter(
+      (el) =>
+        !el.isDeleted ||
+        // Drop deleted elements older than a day
+        el.updated > getUpdatedTimestamp() - DELETED_ELEMENT_TIMEOUT,
```

Comment:
```suggestion
        el.updated > getUpdatedTimestamp() - DELETED_ELEMENT_TIMEOUT,
```

HEre we should actually just use `Date.now()` so it's testable. Were you getting any test failures?

Reply:
Done in 9997e72.  I didn't get any test failures, either with `getUpdatedTimestamp()` or with `Date.now()`.